cmd/stat.go

```diff
@@ -144,9 +144,6 @@ func doStat(clnt Client, isRecursive bool, targetAlias, targetURL string) error
 			continue
 		}
 		url := targetAlias + getKey(content)
-		if !isAliasURLDir(targetURL) {
-			url = content.URL.Path
-		}
```

Comment:
`doStat(clnt Client, isRecursive bool, targetAlias, targetURL string)` looks like targetURL is not used ... you can remove this as well.

Reply:
done.